ifopt_core/src/problem.cc

```diff
@@ -97,16 +97,13 @@ Problem::EvaluateCostFunction (const double* x)
 }
 
 Problem::VectorXd
-Problem::EvaluateCostFunctionGradient (const double* x, bool use_finite_difference_approximation)
+Problem::EvaluateCostFunctionGradient (const double* x, bool use_finite_difference_approximation, double epsilon)
```

Comment:
can you add a default value also to this epsilon, so the interface remains backwards compatible? Then feel free to merge this PR.

Reply:
I may be missing something, but it has a default value defined in the .h file. Is that what you are wanting?

Reply:
Ah, that's it, missed it. Yeah, feel free to merge and if you need to update distros. Thanks so much!🙏

Reply:
I have release the latest on both noetic and foxy.